CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.8] - 2023-07-25
+
+### Changed
```

Comment:
The `[0.2.8]` header will be added automatically by the workflow, you can remove it here and list the changes under the `unreleased` header:
```suggestion
### Changed
```

Reply:
aha, Thanks for mentioning